test/xdg_desktop_portal_test.dart

```diff
@@ -35,6 +35,29 @@ class MockAccountDialog {
   String toString() => '$runtimeType($parentWindow, $options)';
 }
 
+class MockCamera {
+  final Map<String, DBusValue> options;
+
+  MockCamera(this.options);
+
+  @override
+  int get hashCode => Object.hash(
+      '',
```

Comment:
This is not required - just return the result of `Object.hashAll`

Reply:
Done